### PR TITLE
1.3.0 Release

### PR DESCRIPTION
**Pull request by abusi**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Maintainers and Contributors
+============================
+
+Maintainers
+-----------
+
+* [Aurélien BUSI](mailto:aurelien.busi@numberly.com) - [@abusi](https://github.com/abusi)
+* [Klemen SEVER](mailto:klemen.sever@dailymotion.com) - [@achedeuzot](https://github.com/achedeuzot)
+* [Stanislas CHOLLET](mailto:stanislas.chollet@gmail.com) - [@tsunammis](https://github.com/tsunammis)
+* [Maximilien RAULIC](mailto:maximilien.raulic@dailymotion.com) - [@Maximilien-R](https://github.com/Maximilien-R)
```
Comment (Maximilien-R) [inline CONTRIBUTORS.md:10]: Could you replace it with my personal email instead? 🙏 
```suggestion
* [Maximilien RAULIC](mailto:maximilien.raulic@gmail.com) - [@Maximilien-R](https://github.com/Maximilien-R)
```

File: changelogs/1.3.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# [1.2.0] - 2020-07-31
```
Comment (Maximilien-R) [inline changelogs/1.3.0.md:1]:
```suggestion
# [1.3.0] - 2020-07-31
```

File: changelogs/1.3.0.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+     pass
+```
+
+This is done via the `register_graphql_handler(..., response_formatter= a_method)`.
+
+## Changed
+
+* [ISSUE-94](https://github.com/tartiflette/tartiflette-aiohttp/issues/94) - Changed the context_factory from a simple async method to an asynccontextmanager decorated one - Thanks [@jugdizh](https://github.com/jugdizh)
+
+* Test platform:
```
Comment (Maximilien-R) [inline changelogs/1.3.0.md:18]: To be exhaustive, `pylint` has been bumped too from from `2.4.4` to `2.5.3 ` 👍

**Comment by abusi** [comment]
Thanks for the review @Maximilien-R

**Pull request merged**